galsim/roman/roman_psfs.py

```diff
@@ -20,36 +20,37 @@
 import os
 
 """
-@file wfirst_psfs.py
+@file roman_psfs.py
 
-Part of the WFIRST module.  This file includes routines needed to define a realistic PSF for WFIRST.
+Part of the Roman Space Telescope module.  This file includes routines needed to define a realistic
+PSF for Roman.
 """
 
 # Define a default set of bandpasses for which this routine works.
 default_bandpass_list = ['J129', 'F184', 'W149', 'Y106', 'Z087', 'H158']
 # Prefix for files containing information about Zernikes for each SCA for cycle 7.
-zemax_filepref = "WFIRST_Phase-A_SRR_WFC_Zernike_and_Field_Data_170727"
+zemax_filepref = "Roman_Phase-A_SRR_WFC_Zernike_and_Field_Data_170727"
 zemax_filesuff = '.txt'
 zemax_wavelength = 1293. #nm
 
 def getPSF(SCA, bandpass,
            SCA_pos=None, approximate_struts=False, n_waves=None, extra_aberrations=None,
            logger=None, wavelength=None, high_accuracy=False, gsparams=None):
-    """Get a single PSF for WFIRST observations.
+    """Get a single PSF for Roman ST observations.
 
     The user must provide the SCA and bandpass; the latter is used when setting up the pupil
     plane configuration and when interpolating chromatic information, if requested.
 
     This routine carries out linear interpolation of the aberrations within a given SCA, based on
-    the WFIRST Cycle 7 specification of the aberrations as a function of focal plane position, more
-    specifically from a file `WFIRST_Phase-A_SRR_WFC_Zernike_and_Field_Data_170727.xlsm` downloaded
-    from https://wfirst.gsfc.nasa.gov/science/WFIRST_Reference_Information.html.  Phase B updates
-    that became available in mid-2019 have not yet been incorporated into this module.
+    the Roman (then WFIRST) Cycle 7 specification of the aberrations as a function of focal plane
+    position, more specifically from `Roman_Phase-A_SRR_WFC_Zernike_and_Field_Data_170727.xlsm`
+    downloaded from https://roman.gsfc.nasa.gov/science/Roman_Reference_Information.html.  Phase
```

Comment:
I'm not sure how much this matters, but I noticed the files at that URL still are named WFIRST (i.e., the WFIRST->Roman happened in the URL for the page, but not for any of the files downloaded from there).

Reply:
Thanks.  I checked the urls, but I didn't look at the files hosted there.  I added a caveat about this in the doc.